pkg/kubelet/eviction/helpers.go

```diff
@@ -1258,8 +1258,8 @@ func evictionMessage(resourceToReclaim v1.ResourceName, pod *v1.Pod, stats stats
 				var usage *resource.Quantity
 				switch resourceToReclaim {
 				case v1.ResourceEphemeralStorage:
-					if containerStats.Rootfs != nil && containerStats.Rootfs.UsedBytes != nil && containerStats.Logs != nil && containerStats.Logs.UsedBytes != nil {
-						usage = resource.NewQuantity(int64(*containerStats.Rootfs.UsedBytes+*containerStats.Logs.UsedBytes), resource.BinarySI)
+					if containerStats.Rootfs != nil && containerStats.Rootfs.UsedBytes != nil && containerStats.Logs != nil && containerStats.Logs.UsedBytes != nil && podStats.EphemeralStorage.UsedBytes != nil {
+						usage = resource.NewQuantity(int64(*containerStats.Rootfs.UsedBytes + *containerStats.Logs.UsedBytes + *podStats.EphemeralStorage.UsedBytes), resource.BinarySI)
```

Comment:
Does it make sense to also add inode info to the eviction message? If eviction is triggered by a lack of inodes it could be also not easy to understand for users without inode info in the message.